### ISSUE TITLE
Button after video element with no controls is not accessible

### ISSUE THREAD

**Issue by joe-watkins** [FreedomScientific/VFO-standards-support#78]
### Summary

When a button immediately follows a `<video>` element that has no controls, that button can be reached by JAWS but the accessible name is not announced nor can the control be activated.

(Warning test link has autoplay video with no controls to stop)

1. With JAWS activated, visit [this test test](https://s.codepen.io/joe-watkins/debug/geRgMe) using IE11.
2. Attempt to activate the button that follows the `<video>` element.

### Expected result

1. JAWS will announce the button correctly and will be able to activate the button with enter or spacebar.

### Actual result

1. JAWS does not announce the button's accessible name nor can it activate the control.

### Example

> https://s.codepen.io/joe-watkins/debug/geRgMe

### Additional Information
When controls are present, by adding the `controls` attribute on the `<video>` element the button that follows the control is accessible. I believe this extends to the `<audio>` element as well.

An example use case for a video with no controls and a button that follows it would be a video background for a hero that has a button to stop animation/video.

#### JAWS version and build number
18.0.5038

 
#### Operating System and version
WIN7

#### Browser and version:
IE11

**Comment by clane** [comment]
This issue also occurs with the audio tag. See http://www.chrislane.info/examples/buttonLabelNotAnnouncedDueToAudioTag.html

**Comment by stevefaulkner** [comment]
Tested with JAWS 2018 and can reproduce

**Comment by OwenEdwards** [comment]
This significantly impacts frameworks such as Video.js which provide alternative user controls for video playback. See https://github.com/videojs/video.js/issues/4583

**Comment by stevefaulkner** [comment]
Thanks for the issue report, have filed internally as Bug 101267 - Cannot interact with HTML video element UI. In investigating this issue found some further issues with JAWS/IE video element support.
Using JAWS with Internet Explorer 11, there are various major issues with
interacting with HTMl5 video elements:

> 1. When the video element does not have the controls attribute set, a <button>
> placed after the <video> element is not announced when tabbed to.
> 2. When the video element does have the controls attribute when the video
> element receives focus via tabbing, the accessible name announced comes from
> elements surrounding the video element, despite the video element having a
> explicitly set name (via aria-label, for example). Sometimes when tabbing
> through the video UI elements the same accessible name is used for all of the
> controls at other times nothing is announced at other times the full string for
> all video UI is announced for each of the individual controls.

> to reproduce: navigate to https://s.codepen.io/stevef/debug/LmdNga
> tab through the page:
> Note on test 1 the button after the video is not announced
> Note on test 2 sometimes nothing is announced or the each control has the same accessible name or the accessible name is from the button after or the heading before.cannot appear to operate video UI controls.

In investigating the original issue I found that the IE/JAWS issue with the button not being announced can be mitigated by placing `role="application"` on the `video` element. test case https://s.codepen.io/stevef/debug/LmdNga which may be helpful in the short term.

**Comment by stevefaulkner** [comment]
@OwenEdwards see my comment about use of `role=application` which may be a short term fix, until JAWS itself is fixed.

**Comment by joe-watkins** [comment]
@stevefaulkner Thanks for digging into this and for the update w/temp fix :)

**Comment by OwenEdwards** [comment]
@stevefaulkner this issue appears to **also** affect the latest version of Chrome (Version 72.0.3626.109 (Official Build) (64-bit)) and JAWS (2019.1901.66); see @joe-watkins' examples at:

**Without `<video>` element native controls:** _(JAWS+Chrome has trouble with button)_
https://s.codepen.io/joe-watkins/debug/b343f69dddba68cc6f5427e1e869e72e
 
**With `<video>` element native controls:** _(JAWS+Chrome appears to access button)_
https://s.codepen.io/joe-watkins/debug/d2fb16335b121511a9eb1e60e779cea0

I think this is a *new* problem, because we hadn't run into it when we initially reported #78 as affecting IE. I don't know if it is due to a change in Chrome, in JAWS, or in both.

Joe was also able to confirm that the previous workaround seems to work (adding role="application" on the `<video>` element), but it's alarming if this bug is getting worse or more widespread!

**Comment by hadleyel** [comment]
I have tested the linked examples with JAWS 2020.2008.24, Chrome 85.0.4183.121, and IE 11.508.19041.0, and the buttons seem to announce and operate properly now. Is this true for anyone else?

**Comment by JAWS-test** [comment]
With JAWS 2020, the problem no longer occurs in Chrome and IE 11, but continues to occur with older versions of JAWS (for example, JAWS 2019 and IE 11 or JAWS 18 and Chrome)

**Comment by stevefaulkner** [comment]
as the issue is no longer present in 2020, am closing this issue.

**Comment by OwenEdwards** [comment]
@stevefaulkner are you (and @JAWS-test) confirming that this issue has been fixed in JAWS? Just wondering how this process works, since I see you had previously labeled this issue as "JAWS-bug-filed" - are you able to confirm back from that (internal) bug that this was fixed in a specific version of JAWS, or will it always be a matter of people noticing that the bug is no longer reproducible?

**Comment by JAWS-test** [comment]
@OwenEdwards: Unfortunately I cannot answer the question about the internal process because I have nothing to do with Freedom Scientific. I only publish JAWS bugs that I find in my work and comment on JAWS bugs that other people have posted

**Comment by OwenEdwards** [comment]
@JAWS-test ahh, my mistake. Then this is just a question for @stevefaulkner.
